Modules/Filter/include/itkSharedPointerDataObjectDecorator.hxx

```diff
@@ -48,7 +48,7 @@ SharedPointerDataObjectDecorator< T >
 template< typename T >
 void
 SharedPointerDataObjectDecorator< T >
-::Set( std::shared_ptr< T > val )
+::Accept( std::shared_ptr< T > val )
```

Comment:
Are you sure about renaming SharedPointerDataObjectDecorator::Set(std::shared_ptr)? It seems like a different kind of function (non-virtual, returning void).

Reply:
(Oops, I was accidentally using our Jenkins account!)

Reply:
Your mistake actually proves the point of renaming the other Set functions  :-)

Reply:
The SharedPointerDataObjectDecorator is not used anymore. I will remove the h and hxx files.